--- wicd-1.7.0/in/init=lunar=wicd.in~	2010-01-15 06:49:11.000000000 +0200
+++ wicd-1.7.0/in/init=lunar=wicd.in	2010-01-19 09:37:35.202503440 +0200
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/bin/sh
 #
 # Startup script for wicd
 #
--- wicd-1.7.0/in/scripts=wicd-cli.in~	2010-01-15 06:49:11.000000000 +0200
+++ wicd-1.7.0/in/scripts=wicd-cli.in	2010-01-19 09:37:49.590000593 +0200
@@ -1,2 +1,2 @@
-#!/bin/bash
+#!/bin/sh
 exec %PYTHON% -O %SHARE%cli/wicd-cli.py $@
