--- wicd-1.7.0/other/wicd.desktop~	2010-01-15 05:49:11.000000000 +0100
+++ wicd-1.7.0/other/wicd.desktop	2010-05-12 16:10:21.505141573 +0200
@@ -1,5 +1,5 @@
 [Desktop Entry]
-Categories=Application;Network;
+Categories=Application;Network;System;
 Exec=wicd-gtk --no-tray
 GenericName=Network Manager
 Icon=wicd-gtk
--- wicd-1.7.0/other/wicd-tray.desktop~	2010-01-15 05:49:11.000000000 +0100
+++ wicd-1.7.0/other/wicd-tray.desktop	2010-05-12 16:10:30.895203879 +0200
@@ -1,5 +1,5 @@
 [Desktop Entry]
-Categories=Application;Network;
+Categories=Application;Network;System;
 Exec=wicd-gtk --tray
 GenericName=Network Manager
 Icon=wicd-gtk
